presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -169,6 +169,8 @@ Property Name                                      Description
                                                    See :ref:`hive-file-based-authorization` for details.
 
 ``hive.non-managed-table-writes-enabled``          Enable writes to non-managed (external) Hive tables.         ``false``
+
+``hive.create-non-managed-table-enabled``          Enable creating non-managed (external) Hive tables.          ``true``
```

Comment:
One can also `create schema with (location='s3://some/location')` and then try to create table that matches existing dir under `s3://some/location`. Is this path to query arbitrary existing S3 data also somehow prevented?